packages/nala/build.sh

```diff
@@ -2,23 +2,21 @@ TERMUX_PKG_HOMEPAGE=https://gitlab.com/volian/nala
 TERMUX_PKG_DESCRIPTION="Commandline frontend for the apt package manager"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.7.2
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION=0.11.1
 TERMUX_PKG_SRCURL=https://gitlab.com/volian/nala/-/archive/v${TERMUX_PKG_VERSION}/nala-v${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=0ac394ab2bd924c28c513a8e8bfc9e899a21f2f8fa5e5d77366cb20e0be8676e
+TERMUX_PKG_SHA256=4ce87d75e785e45b1638817077797a3fcd4a15059615455fc1af0127c70cf5cd
 TERMUX_PKG_DEPENDS="python-apt"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
 
-_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION)
+termux_step_pre_configure() {
+	_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION)
 
-termux_step_post_get_source() {
-	sed 's:@TERMUX_PKG_VERSION@:'${TERMUX_PKG_VERSION##*:}':g' \
-		${TERMUX_PKG_BUILDER_DIR}/pyproject.toml.diff | \
-		patch --silent -p1
-}
+	TERMUX_PKG_RM_AFTER_INSTALL="
```

Comment:
@Grimler91 shouldn't this 2 variables have been exported? Since they are now restricted to this function

Reply:
It works as is, basically our build.sh script is equivalent to:

```
$ cat build.sh
foo() {
    _PYTHON_VERSION=123
}
bar() {
    echo "something something $_PYTHON_VERSION"
}
```

and `build-package.sh` then runs

```
source build.sh
foo
bar
```

which echos `something something 123`, so `_PYTHON_VERSION` is set in bar() even though it is not exported in foo().

export would be needed if we create a new subshell after sourcing the build.sh, without export this echoes nothing:

```
source build.sh
foo
bash -c 'echo $_PYTHON_VERSION'
```

Reply:
Sometimes I forget how unintuitive bash can be sometimes